### ISSUE TITLE
first impressions

### ISSUE THREAD

**Issue by bennn** [default-kramer/fission-flare#1]
This looks fun but I haven't figured out how to play!

First thing I did was open the game & start pressing buttons. I saw that the arrow keys moved the pill around, so I moved it to the bottom. The pill got stuck at the bottom and then NOTHING else I pressed seemed to do anything.

Second, I read the docs and learned that left and right shift are important. Sometimes holding right shift didn't do anything, but toggling left & right shift made a pill drop. After a few moves, the game crashed.

Here's the console log:
```
[1]  - continued  racket main-frame.rkt
'("re-anchor:" 1632443856670.269)
'("re-anchor:" 1632443896413.852)
Failure on line 99 of frame.rkt ("frame burst failed")
  context...:
   /Users/ben/code/racket/fork/extra-pkgs/fission-flare/src/ui/controller.rkt:68:9
   /Users/ben/code/racket/fork/extra-pkgs/fission-flare/src/ui/main-frame.rkt:67:2: loop
   /Users/ben/code/racket/fork/extra-pkgs/gui/gui-lib/mred/private/wx/common/queue.rkt:435:6
   /Users/ben/code/racket/fork/extra-pkgs/gui/gui-lib/mred/private/wx/common/queue.rkt:486:32
   /Users/ben/code/racket/fork/extra-pkgs/gui/gui-lib/mred/private/wx/common/queue.rkt:634:3
```

And here's a screenshot:
![Screen Shot 2021-09-23 at 20 39 06](https://user-images.githubusercontent.com/1731829/134600823-9116f66a-b6ae-420b-b394-18bcdcdd170c.png)

**Comment by default-kramer** [comment]
Oh yeah, I forgot about that bug! I definitely meant to fix that one before release! Thanks.

**Comment by default-kramer** [comment]
... And the up and down arrow keys are not supposed to do anything. Forgot about that too.

**Comment by default-kramer** [comment]
The unintended behavior is fixed in version 0.2. Please let me know if you are still confused, because yeah, the documentation is rather light, and gamers don't want to read documentation anyway. So I probably should still do something to make that better.

**Comment by bennn** [comment]
Much better!

New question:
- if I hit enter/return
- and then try right-shift, it doesn't do anything
- until I hit left-shift --- after that (and only that), right-shift works again
Is this a bug?

**Comment by default-kramer** [comment]
Yes, that sounds like a bug. Which OS are you using?

**Comment by default-kramer** [comment]
I suspect this is fixed in Version 0.5 (d6e71353dbd53e0d00d71e0e7911caca6455c4db)

**Comment by bennn** [comment]
yes, much better!

I'm on a mac --- OSX 10.11.1
